File: Source/Rio.Web/src/app/shared/models/parcel-allocation-type-dto.ts
export class ParcelAllocationTypeDto {
    ParcelAllocationTypeID: number;
    ParcelAllocationTypeName: string;
    IsAppliedProportionally: ParcelAllocationTypeApplicationTypeEnum;
    ParcelAllocationTypeDefinition: string;
    SortOrder: number;

    constructor(obj?: any){
        Object.assign(this, obj);
    }
}

export enum ParcelAllocationTypeApplicationTypeEnum{
    Spreadsheet = 0,
    Proportional = 1,
    Api = 2
}

File: Source/Rio.Models/DataTransferObjects/ParcelAllocationTypeDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Rio.Models.DataTransferObjects
{
    public class ParcelAllocationTypeDto
    {
        public int ParcelAllocationTypeID { get; set; }
        public string ParcelAllocationTypeName { get; set; }
        public ParcelAllocationTypeApplicationTypeEnum IsAppliedProportionally { get; set; }
        public string ParcelAllocationTypeDefinition { get; set; }
        public int SortOrder { get; set; }
    }

    public enum ParcelAllocationTypeApplicationTypeEnum
    {
        Spreadsheet = 0,
        Proportionally = 1,
        Api = 2
    }
}
